helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -62,10 +59,33 @@ controller:
   ##
   minReadySeconds: 10
 
+
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: {{- toYaml .Values.ingressNginx.controller.tolerations | nindent 4 }}
+
+  ## Affinity and anti-affinity
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {{- toYaml .Values.ingressNginx.controller.affinity | nindent 4 }}
+
   ## terminationGracePeriodSeconds
+  ## wait up to five minutes for the drain of connections
   ##
   terminationGracePeriodSeconds: 60
 
+  ## Node labels for controller pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {{- toYaml .Values.ingressNginx.controller.nodeSelector | nindent 4 }}
+
+  # Define requests resources to avoid probe issues due to CPU utilization in busy nodes
+  # ref: https://github.com/kubernetes/ingress-nginx/issues/4735#issuecomment-551204903
+  # Ideally, there should be no limits.
+  # https://engineering.indeedblog.com/blog/2019/12/cpu-throttling-regression-fix/
+  resources: {{- toYaml .Values.ingressNginx.controller.resources | nindent 4 }}
+
   service:
     {{ if .Values.externalTrafficPolicy.local }}
     ## Set external traffic policy to: "Local" to preserve source IP on
```

Comment:
Check this whole `service` block, it seems to have changed a bit.

Reply:
I check, but didn't notice any change. What did you see as NOK?

Reply:
Re-checked now and can't see anything. Must have misread earlier.

Reply:
ok